app/controllers/dashboard.js

```diff
@@ -0,0 +1,59 @@
+const Departments = require('../models/departments');
+const Users = require('../models/users');
+const Messages = require('../models/messages');
+
+module.exports = {
+
+  orgIndex(req, res) {
```

Comment:
Then also updated line 172 `app.get("/org", DashboardController.org);` in app.js